specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2025-06-01-preview/examples/ClusterCreate.json

```diff
@@ -13,7 +13,7 @@
       "properties": {
         "administratorLogin": "examplelogin",
         "administratorLoginPassword": "examplepassword",
-        "version": "16",
+        "version": "17",
```

Comment:
In portal we get "PostgreSQL version 17 is currently not supported for elastic cluster". Can you confirm if this is allowed now?